.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
I would turn this into something more specific with examples, something like:

Please include a summary of the change, motivation and context

- on a feature: describe the feature and how this change fits in it, e.g. *this PR makes kafka `message.max.bytes` configurable to better support batching*
- on a refactor: describe why this is better than previous situation e.g. *this PR changes logic for retry on healthchecks to avoid false positives*
- on a bugfix: link relevant information about the bug (github issue or slack thread) and how this change solves it e.g. *this change fixes #99999 by adding a lock on read/write to avoid data races.*

Reply:
addressed! Please check!